dev-libs/spdlog/spdlog-1.4.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="Very fast, header only, C++ logging library"
+HOMEPAGE="https://github.com/gabime/spdlog"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/gabime/${PN}"
+else
+	SRC_URI="https://github.com/gabime/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0/1"
+IUSE="static-libs test"
```

Comment:
`static-libs` is new, is there any reason to have that? If not, please drop it.

Reply:
It was added in the PR made by @lovesegfault, I don't know if he has some special reason, at the moment I don't see any so I dropped it as requested.

Reply:
I'm fine with dropping it :)